<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NeuralWindow</class>
 <widget class="QMainWindow" name="NeuralWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1920</width>
    <height>1080</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Movie and TV series recommendation system</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>C:/Users/Lili/.designer/images/video.png</normaloff>C:/Users/Lili/.designer/images/video.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(189, 205, 214);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout" columnstretch="1,1" columnminimumwidth="2,1">
    <property name="sizeConstraint">
     <enum>QLayout::SetDefaultConstraint</enum>
    </property>
    <property name="leftMargin">
     <number>30</number>
    </property>
    <property name="topMargin">
     <number>30</number>
    </property>
    <property name="rightMargin">
     <number>30</number>
    </property>
    <property name="bottomMargin">
     <number>30</number>
    </property>
    <property name="horizontalSpacing">
     <number>30</number>
    </property>
    <item row="0" column="0">
     <layout class="QVBoxLayout" name="verticalLayout">
      <property name="spacing">
       <number>50</number>
      </property>
      <property name="leftMargin">
       <number>50</number>
      </property>
      <property name="topMargin">
       <number>50</number>
      </property>
      <property name="rightMargin">
       <number>50</number>
      </property>
      <property name="bottomMargin">
       <number>50</number>
      </property>
      <item>
       <widget class="QPushButton" name="ButtonLoadModel">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>100</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>Rockwell</family>
          <pointsize>30</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(243, 222, 186);
border-radius: 20px;</string>
        </property>
        <property name="text">
         <string>Load model</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="ButtonMakeModel">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>100</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>Rockwell</family>
          <pointsize>30</pointsize>
          <weight>50</weight>
          <bold>false</bold>
          <strikeout>false</strikeout>
         </font>
        </property>
        <property name="layoutDirection">
         <enum>Qt::LeftToRight</enum>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(243, 222, 186);
border-radius: 20px;</string>
        </property>
        <property name="text">
         <string>Make new model</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="ButtonSaveModel">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>100</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>Rockwell</family>
          <pointsize>30</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(243, 222, 186);
border-radius: 20px;</string>
        </property>
        <property name="text">
         <string>Save model</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="ButtonTrain">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>100</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>Rockwell</family>
          <pointsize>30</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(243, 222, 186);
border-radius: 20px;</string>
        </property>
        <property name="text">
         <string>Train</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="labelAccuracy">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="font">
         <font>
          <family>Rockwell</family>
          <pointsize>48</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(255, 255, 255);
padding: 5px;</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="ButtonGoToRecommendation">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>100</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>Rockwell</family>
          <pointsize>30</pointsize>
          <weight>50</weight>
          <bold>false</bold>
          <strikeout>false</strikeout>
         </font>
        </property>
        <property name="layoutDirection">
         <enum>Qt::LeftToRight</enum>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(243, 222, 186);
border-radius: 20px;</string>
        </property>
        <property name="text">
         <string>Give recommendations</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="0" column="1">
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <property name="spacing">
       <number>20</number>
      </property>
      <property name="sizeConstraint">
       <enum>QLayout::SetMaximumSize</enum>
      </property>
      <property name="leftMargin">
       <number>50</number>
      </property>
      <property name="topMargin">
       <number>50</number>
      </property>
      <property name="rightMargin">
       <number>50</number>
      </property>
      <property name="bottomMargin">
       <number>50</number>
      </property>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1920</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
